apps/admin/integration-testing/playwright.config.ts

```diff
@@ -22,6 +25,7 @@ export default defineConfig({
     ['html', { open: 'never' }],
     ['junit', { outputFile: join(outputDir, 'results.xml') }],
   ],
+  workers: 1,
```

Comment:
Test suites still run in parallel with current settings, we have to only use one worker to give each test exclusive access to app state.

We could instead spin up different app instances for different tests, but that seems like trouble (and resource-intensive).